source/_components/device_tracker.mikrotik.markdown

```diff
@@ -30,37 +32,38 @@ Go to **IP** -> **Services** -> **API** and enable it.
 
 Make sure that port 8728 or the port you choose is accessible from your network.
 
-To use a Mikrotik router in your installation, add the following to your `configuration.yaml` file:
+
+To use a MikroTik router in your installation, add the following to your `configuration.yaml` file:
 
 ```yaml
 # Example configuration.yaml entry
 device_tracker:
   - platform: mikrotik
     host: IP_ADDRESS
-    username: ADMIN_USERNAME
-    password: ADMIN_PASSWORD
+    username: ROUTEROS_USERNAME
+    password: ROUTEROS_PASSWORD
 ```
 
 {% configuration %}
 host:
-  description: The IP address of your router.
+  description: The IP address of your MikroTik device.
   required: true
   type: string
 username:
-  description: The username of an user with administrative privileges.
+  description: The username of a user on the MikroTik device.
   required: true
   type: string
 password:
-  description: The password for your given admin account.
+  description: The password of the given user account on the MikroTik device.
   required: true
   type: string
 port:
-  description: Mikrotik API port.
+  description: RouterOS API port.
   required: false
   default: 8728 (or 8729 if ssl is true)
   type: integer
 ssl:
-  description: Use api_ssl service instead of api.
+  description: Use api-ssl service instead of API.
```

Comment:
api and api-ssl are names of services in RouterOS in lowercase.
See: /ip service print

Reply:
There are the API for for developers, and there are api and api-ssl services.

Reply:
API can be accessed via api or api-ssl services.

Reply:
How about "Use SSL to connect to the API."

Reply:
Good! I updated the PR.